tests/test_causal_problem/test_callables.py

```diff
@@ -94,7 +94,7 @@ def which(request: pytest.FixtureRequest) -> Literal["causal_estimand", "constra
             0.0,
             # Empirical calculation with 1000 samples with fixture RNG key
             # should give 1.8808e-2 as the empirical expectation.
-            2.0e-2,
+            3.0e-2,
```

Comment:
These tests started failing, I think due to the rng keys being different in the new way of doing things. Slightly upping the tolerances fixed it. If you don't agree that this is ok, we should look into what I broke

Reply:
Yeah this will do it - we're no longer splitting the key each time we draw a single sample from the distribution (since it's now all done in parallel).